bftengine/src/bftengine/KeyManager.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "KVBCInterfaces.h"
+#include "Serializable.h"
+
+class KeyManager {
+ public:
+  struct KeyExchangeMsg : public concord::serialize::SerializableFactory<KeyExchangeMsg> {
+    std::string key_;
+    std::string signature_;
+    uint32_t repID_;
+    std::string toString() const;
+    static KeyExchangeMsg MsgDeserializer(const char* serStr, const uint32_t& size);
+
+   protected:
+    const std::string getVersion() const;
```

Comment:
`const` for the return value is redundant.

Reply:
It's required from the Serializable interface

Reply:
Holy crp, gotcha!